homeassistant/components/lektrico/sensor.py

```diff
@@ -0,0 +1,296 @@
+"""Support for Lektrico charging station sensors."""
+
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from typing import Any
+
+from lektricowifi import Device
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_FRIENDLY_NAME,
+    PERCENTAGE,
+    UnitOfElectricCurrent,
+    UnitOfElectricPotential,
+    UnitOfEnergy,
+    UnitOfPower,
+    UnitOfTemperature,
+    UnitOfTime,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import PlatformNotReady
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from . import LektricoDeviceDataUpdateCoordinator
+from .const import DOMAIN
+from .entity import LektricoEntity
+
+
+@dataclass(frozen=True, kw_only=True)
+class LektricoSensorEntityDescription(SensorEntityDescription):
+    """A class that describes the Lektrico sensor entities."""
+
+    value_fn: Callable[[Any], float | str | int]
+
+
+SENSORS_FOR_CHARGERS: tuple[LektricoSensorEntityDescription, ...] = (
+    LektricoSensorEntityDescription(
+        key="state",
+        translation_key="state",
+        value_fn=lambda data: str(data.charger_state),
+    ),
+    LektricoSensorEntityDescription(
+        key="charging_time",
+        translation_key="charging_time",
+        native_unit_of_measurement=UnitOfTime.SECONDS,
+        value_fn=lambda data: int(data.charging_time),
+    ),
+    LektricoSensorEntityDescription(
+        key="power",
+        device_class=SensorDeviceClass.POWER,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfPower.KILO_WATT,
+        value_fn=lambda data: float(data.instant_power) / 1000,
```

Comment:
```suggestion
        native_unit_of_measurement=UnitOfPower.WATT,
        suggested_unit_of_measurement=UnitOfPower.KILO_WATT
        value_fn=lambda data: float(data.instant_power),
```
This is the same but utilizes HA to do the conversion

Reply:
I changed it, thanks.